src/python/pants/core/goals/test.py

```diff
@@ -46,20 +46,25 @@ class TestResult:
     status: Status
     stdout: str
     stderr: str
-    coverage_data: Optional["CoverageData"] = None
+    coverage_data: Optional["CoverageData"]
+    xml_test_results: Optional[Digest]
```

Comment:
Maybe call it `xml_results`? I think the `test` part is obvious enough since the property lives on a `TestResult` class.